ansible/roles/csm.sbps.dns_srv_records/files/sbps_dns_srv_records.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+set -euo pipefail
+
+PDNS_API_KEY=$(kubectl -n services get secret cray-powerdns-credentials -o jsonpath='{.data.pdns_api_key}' | base64 -d)
+PDNS_API=$(kubectl -n services get svc cray-dns-powerdns-api -o jsonpath='{.spec.clusterIP}')
+
+hsn_srv_records=""
+nmn_srv_records=""
+hsn_a_records=""
+nmn_a_records=""
+
+system_name="$(cat /etc/cray/system_name)"
```

Comment:
I don't think this is reliable, I don't see `/etc/cray/system_name` on mug:
```
ncn-m001:~ # ls /etc/cray
kubernetes  nid  upgrade  xname
ncn-m001:~ # 
```
and I don't see it on baldar:
```
ncn-m001:~ # ls /etc/cray
ca  kubernetes	nid  upgrade  xname
```
Though it does exist on odin:
```
ncn-m001:~ # ls /etc/cray
kubernetes  nid  system_name  upgrade  xname
ncn-m001:~ # 
```
This seems a bit kludgy, but it should work, assuming that CSM has already been installed:
```
ncn-m001:~ # kubectl get vs -n sysmgmt-health cray-sysmgmt-health-grafana -o jsonpath='{.spec.hosts[0]}' | awk -F . '{print $3}'
mug
ncn-m001:~ # 
```

Reply:
It looks like you can just use ${SYSTEM_NAME}, which is defined in `/etc/environment`
```
ncn-m001:~ # cat /etc/environment 
#
# This file is parsed by pam_env module
#
# Syntax: simple "KEY=VAL" pairs on seperate lines
#
SYSTEM_NAME=mug
SITE_DOMAIN=hpc.amslabs.hpecorp.net
SYSTEM_DOMAIN=mug.hpc.amslabs.hpecorp.net
```